### PR TITLE
[SYCL][NFC] Temporarily disable Driver/sycl.c test on Windows

### PR DESCRIPTION
**Pull request by mdtoguchi**
Test is failing - disable for now until we can find a fix.

### CODE REVIEW

**Review by steffenlarsen — APPROVED** [review]
Thanks a ton, @mdtoguchi ! Could I bother you to also open a JIRA tracker for it, just so we don't forget it?

**Comment by mdtoguchi** [comment]
> Thanks a ton, @mdtoguchi ! Could I bother you to also open a JIRA tracker for it, just so we don't forget it?

JIRA has been submitted, I would hope to have a real fix in the near term.

File: clang/test/Driver/sycl.c [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Failing on Windows - temporarily disable
+// REQUIRES: system-linux
```
Comment (bader) [inline clang/test/Driver/sycl.c:2]: I don't suggest making any changes to this patch, but I have a general recommendation.
The better way to disable the test on one platform is to mark is as unsupported on that platform (e.g. `UNSUPPORTED: system-windows`). 
As of now DPC++ is validated on three different platforms Windows, Linux and MacOS. MacOS validation doesn't cover clang lit tests, so existing approach provides the desired result, but potentially reduces the testing scope on supported platforms.

**Review by hchilama — APPROVED** [review]

**Comment by mdtoguchi** [comment]
@intel/llvm-gatekeepers, this PR is ready to go - thanks!

**Pull request merged**